### PR TITLE
CMake: ddr bug fixes

### PR DESCRIPTION
**Pull request by dnakamura**
See individual commits for detailed descriptions

### CODE REVIEW

**Comment by charliegracie** [comment]
@genie-omr build all

**Comment by charliegracie** [comment]
@keithc-ca does this look ok to you?

File: cmake/modules/ddr/DDRSetStub.cmake.in [hunk 1]
```diff
@@ -74,7 +74,11 @@ function(process_source_files src_files)
 
 
 	foreach(source_file IN LISTS OPT_UNPARSED_ARGUMENTS)
-
+		get_filename_component(extension "${source_file}" EXT)
+		# If file isnt a c/c++ file, ignore it
```
Comment (keithc-ca) [inline cmake/modules/ddr/DDRSetStub.cmake.in]: nit: spelling and punctuation - "isnt" -> "isn't"; missing period.

**Comment by keithc-ca** [comment]
> @keithc-ca does this look ok to you?

I'm not a cmake expert, but it looks good to me.

**Pull request merged**